python/nano/src/bigdl/nano/deps/openvino/core/model.py

```diff
@@ -148,3 +166,37 @@ def pot(self,
             model = Core().read_model(model_path)
             model.reshape(orig_shape)
         return model
+
+    def async_predict(self, input_data, num_infer_requests=0):
+        """
+        Perfrom model inference using async mode.
+
+        :param input_data: Input data to be inferenced.
+                           Users can put multiple input data in a list to infer them together.
+                           Can be List[numpy.ndarray] or
+                           List[List[numpy.ndarray]] if the model has multiple inputs.
+        :param num_infer_requests: Number of infer requests in the AsyncInferQueue, default to 0.
```

Comment:
I don't think the user will understand `AsyncInferQueue`; instead, explain it's a pool of asynchronous  interference requests.